tdclient/table_api.py

```diff
@@ -132,3 +134,33 @@ def delete_table(self, db, table):
             js = self.checked_json(body, [])
             t = js.get("type", "?")
             return t
+
+    def tail(self, db, table, count, to=None, _from=None, block=None):
+        """Get the contents of the table in reverse order based on the registered time
+        (last data first).
+
+        Args:
+            db (str): Target database name.
+            table (str): Target table name.
+            count (int): Number for record to show up from the end.
+            to: Deprecated parameter.
+            _from: Deprecated parameter.
+            block: Deprecated parameter.
+
+        Returns:
+            [dict]: Contents of the table.
+        """
+        params = {"count": count, "format": "msgpack"}
+        with self.get(
+            "/v3/table/tail/%s/%s" % (urlquote(str(db)), urlquote(str(table))), params
+        ) as res:
+            code = res.status
+            if code != 200:
+                self.raise_error("Tail table failed", res, "")
+
+            unpacker = msgpack.Unpacker(res, raw=False)
```

Comment:
off topic - might be better tweaking other unpacker code as well to specify `raw=False` in lieu of `encoding="utf-8"`

https://github.com/treasure-data/td-client-python/blob/0.14.0/tdclient/job_api.py#L197
https://github.com/treasure-data/td-client-python/blob/0.14.0/tdclient/test/test_helper.py#L73

Reply:
Agreed. I'll follow it up with another PR.